### PR TITLE
T1 20.landing

### PR DESCRIPTION
**Pull request by Cortesjohnj**
# ADOGTA LANDING PAGE

I've created a fully functional landing page, simple and user friendly with different features.

## Features
  - A simple layout that is mobile friendly, with animations, smooth scroller (every component was done with pure css(different components have animations and smooth transitions).
  - A photo slider at the info section with pure css and mobile friendly.
  
about
aboutmobile1
helpus
helpusmobile
info
infomobile1

  - A mobile friendly adaptative Navbar with different functioning buttons, it changes dynamically if the user exist using ternary operators to change classNames and styles.
  
navbarIsUser
navbarIsUserMobile

  - A mobile friendly adaptative footer with different functioning buttons, it changes dynamically if the user exist using ternary operators to change classNames and styles.
  
footerIsUser
footerIsUserMobile
footermobile
footer

  - A mobile friendly  Sidebar with different functioning buttons, it appears and dissapears depending on the window with, also I used setState hook to know if the sidebar is openend or not and a function to toogle it when it's clicked.
  
sidebar 1

  - A mobile friendly functioning whatsapp button that sends the user to the established phone number.
  
whatsapp
whatsapp2

### CODE REVIEW

File: src/components/Info.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            We connect you with your dream pet
+          </div>
+          <h1 className="infoContainer__wrapper--heading">
+            Sign up, click on foundation card, then select your favorite pet and
+            you´re ready to start the adoption process.
+          </h1>
+          <p className="infoContainer__wrapper--subtitle">
+            Your dream pet is waiting for you
+          </p>
+          {/* <div className="infoContainer__wrapper--btnWrap">
```
Comment (leoneloliveros) [inline src/components/Info.js:27]: Please remove this commented code

**Comment by leoneloliveros** [comment]
Really good use of components, and great indentation. Great work only remove commented code

File: src/components/Info.js (hunk 1 above)
Comment (Cortesjohnj) [reply src/components/Info.js]: Thanks, I've deleted the commented code in info.js and also resolved some css issues and conflicts with develop.

File: src/assets/styles/AboutSection.css [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+}
+
+.aboutContainer__wrapper--textWrapper {
+  max-width: 540px;
+  padding-top: 0;
+  padding-bottom: 60px;
+  margin: 0 44px;
+}
+
+@media (max-width: 768px) {
```
Comment (leoneloliveros) [inline src/assets/styles/AboutSection.css:45]: This media query is duplicated, try to use only one

File: src/assets/styles/HelpUs.css [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+.joinUsContainer__wrapper {
+  z-index: 1;
+  min-height: 100vh;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+@media (max-width: 768px) {
```
Comment (leoneloliveros) [inline src/assets/styles/HelpUs.css:29]: Media query Duplicated

File: src/assets/styles/HelpUs.css [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+  transition: all 0.2s ease-in-out;
+}
+
+.joinUsContainer__wrapper--img {
+  width: 600px;
+  height: 100%;
+  margin: 0 0 10px 0;
+}
+
+@media (max-width: 768px) {
```
Comment (leoneloliveros) [inline src/assets/styles/HelpUs.css:116]: Please use only one media query with 768px and add all selectors related to that size in one media query

File: src/assets/styles/Info.css [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+  margin: 0 44px;
+}
+
+@media (max-width: 952px) {
+  .infoContainer__wrapper--textWrapper {
+    margin-top: 44px;
+  }
+}
+
+@media (max-width: 768px) {
```
Comment (leoneloliveros) [inline src/assets/styles/Info.css:57]: Same here

**Review by leoneloliveros — APPROVED** [review]
Good Job!

**Pull request merged**